commit ccae91e3f2f6936010c1b5e1c3506109623a58ad
Author: Mark Marshall <mark.marshall@omicronenergy.com>
Date:   Thu Jan 26 16:18:27 2017 +0100

    mtd: nand: ifc: Fix location of eccstat registers for IFC V1.0
    
    commit 656441478ed55d960df5f3ccdf5a0f8c61dfd0b3 upstream.
    
    The commit 7a654172161c ("mtd/ifc: Add support for IFC controller
    version 2.0") added support for version 2.0 of the IFC controller.
    The version 2.0 controller has the ECC status registers at a different
    location to the previous versions.
    
    Correct the fsl_ifc_nand structure so that the ECC status can be read
    from the correct location for both version 1.0 and 2.0 of the controller.
    
    Fixes: 7a654172161c ("mtd/ifc: Add support for IFC controller version 2.0")
    Signed-off-by: Mark Marshall <mark.marshall@omicronenergy.com>
    Signed-off-by: Boris Brezillon <boris.brezillon@free-electrons.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1347_ccae91e3f2f6936010c1b5e1c3506109623a58ad.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1347_ccae91e3f2f6936010c1b5e1c3506109623a58ad.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
